resources/views/privacy.blade.php

```diff
@@ -7,11 +7,11 @@
 @endsection
 
 @section('banner')
-<div id="white-wrap" class="py-3">
+<div id="white-wrap" class="pb-3" style="padding-top: calc(72px + 1rem)">
```

Comment:
if you can its better to move this code to CSS file
`style="padding-top: calc(72px + 1rem)"`

Reply:
I changed